Sejima/Sources/MUCollectionPicker/MUCollectionPicker.swift

```diff
@@ -0,0 +1,161 @@
+        (collectionView.collectionViewLayout as? UICollectionViewFlowLayout)?.scrollDirection = scrollDirection
+
+        collectionView.register(MUCollectionPickerCell.self,
+                                forCellWithReuseIdentifier: MUCollectionPickerCell.reusableIdentifier)
+    }
+
+    // MARK: - Public functions
+
+    public func set(_ items: [String]) {
+        self.items = items
+    }
+}
+
+extension MUCollectionPicker: UICollectionViewDataSource {
+    public func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return items.count
+    }
+
+    public func collectionView(_ collectionView: UICollectionView,
+                               cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let reuseIdentifier = MUCollectionPickerCell.reusableIdentifier
+        guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: reuseIdentifier,
+                                                            for: indexPath) as? MUCollectionPickerCell else {
+                                                                return UICollectionViewCell()
+        }
+
+        cell.title = items[indexPath.row]
+        cell.itemSelectedColor = itemSelectedColor
+        cell.itemUnselectedColor = itemUnselectedColor
+        cell.itemFont = itemFont
+        cell.itemAlignment = itemAlignment
+        cell.indicatorHeight = indicatorHeight
+        cell.indicatorWidth = indicatorWidth
+        cell.indicatorColor = indicatorColor
+        cell.indicatorRadius = indicatorRadius
+
+        return cell
+    }
+}
+
+extension MUCollectionPicker: UICollectionViewDelegate {
+    public func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        if let selectedItem = selectedItem {
+            (collectionView.cellForItem(at: selectedItem) as? MUCollectionPickerCell)?.unselect()
+            self.selectedItem = nil
+        }
+
+        if let cell = collectionView.cellForItem(at: indexPath) as? MUCollectionPickerCell {
+            selectedItem = indexPath
+            cell.select()
+        }
+
+        delegate?.didSelect(item: items[indexPath.row], at: indexPath.row)
+    }
+
+    public func collectionView(_ collectionView: UICollectionView,
+                               willDisplay cell: UICollectionViewCell,
+                               forItemAt indexPath: IndexPath) {
+        selectedItem?.row == indexPath.row ?
+            (cell as? MUCollectionPickerCell)?.select() : (cell as? MUCollectionPickerCell)?.unselect()
+    }
+}
+
+extension MUCollectionPicker: UICollectionViewDelegateFlowLayout {
+    public func collectionView(_ collectionView: UICollectionView,
+                               layout collectionViewLayout: UICollectionViewLayout,
+                               sizeForItemAt indexPath: IndexPath) -> CGSize {
+        return delegate?.size(self, at: indexPath) ?? CGSize(width: 50, height: 50)
```

Comment:
Maybe have `defaultWidth` and `defaultHeight` variables to have a behavior without delegate.